src/sentry/static/sentry/app/index.js

```diff
@@ -25,6 +25,11 @@ import ajaxCsrfSetup from 'app/utils/ajaxCsrfSetup';
 import * as il8n from 'app/locale';
 import plugins from 'app/plugins';
 
+// Used for operational metrics to determine that the application js
+// bundle was loaded by browser
+window.__sentryMetrics = window.__sentryMetrics || {};
+window.__sentryMetrics.sentryAppInit = +new Date();
```

Comment:
You could use [window.performance.now()](https://developer.mozilla.org/en-US/docs/Web/API/Performance/now) to get millisecond resolution if you need/want it.

Reply:
but Opera Mobile doesn't support it, so better not :trollface: